cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -127,22 +127,44 @@ ON [EACH] type(r)
 
 Index provider can be specified using the `OPTIONS` clause.
 
+| [source, cypher, role=noplay]
+----
+CREATE TEXT INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName)
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a text index on nodes.
```

Comment:
Should we mention somewhere that text index only handles properties with string values? And if so, where would be good to add that? @Lojjs

Reply:
That make sense yes, maybe we should extend the introduction section `1. Indexes (types and limitations)` with that info?